projects/angular/src/data/tree-view/tree.ts

```diff
@@ -35,30 +35,30 @@ import { ClrTreeNode } from './tree-node';
   host: {
     tabindex: '0',
     '[attr.role]': '"tree"',
-    '[attr.aria-multiselectable]': 'isMultiSelectable',
   },
 })
 export class ClrTree<T> implements AfterContentInit, OnDestroy {
   @ContentChildren(ClrTreeNode) private rootNodes: QueryList<ClrTreeNode<T>>;
 
   private subscriptions: Subscription[] = [];
+  private _isMultiSelectable = false;
 
   constructor(
     public featuresService: TreeFeaturesService<T>,
     private focusManagerService: TreeFocusManagerService<T>,
-    { nativeElement }: ElementRef<HTMLElement>,
```

Comment:
I would just add a `private` modifier and thus use the nativeElement throught the class as a field with less changes but I am fine both ways

Reply:
It won't work. I tested it.

Reply:
@mivaylo should we resolve this comment?